examples/flux-todomvc/README.md

```diff
@@ -238,10 +238,12 @@ export default Todo;
 ```
 
 Now we can use this structure, along with a simple
-[`Counter`](./src/data/Counter.js) to implement the `ADD_TODO` action. Update
+[`Counter`](./src/data/Counter.js) to implement the `ADD_TODO` action. Add the `flux-todomvc/src/data/Counter.js` file to your `my-todomvc/src/data` folder. Then update
```

Comment:
Use a link to the file, and not full path name like how Counter is referenced earlier. I believe this change clears it up and is shorter:

```
Now we can use this structure, along with a simple
[`Counter`](./src/data/Counter.js) to implement the `ADD_TODO` action. Create
or copy [`Counter`](./src/data/Counter.js) then update `data/TodoStore.js`
```